connect/runtime/src/test/java/org/apache/kafka/connect/integration/RebalanceSourceConnectorsIntegrationTest.java

```diff
@@ -350,16 +350,14 @@ private boolean assertConnectorAndTasksAreUniqueAndBalanced() {
             log.debug("Connector balance: {}", formatAssignment(connectors));
             log.debug("Task balance: {}", formatAssignment(tasks));
 
-            assertNotEquals("Found no connectors running!", maxConnectors, 0);
-            assertNotEquals("Found no tasks running!", maxTasks, 0);
-            assertEquals("Connector assignments are not unique: " + connectors,
-                    connectors.values().size(),
-                    connectors.values().stream().distinct().count());
-            assertEquals("Task assignments are not unique: " + tasks,
-                    tasks.values().size(),
-                    tasks.values().stream().distinct().count());
-            assertTrue("Connectors are imbalanced: " + formatAssignment(connectors), maxConnectors - minConnectors < 2);
-            assertTrue("Tasks are imbalanced: " + formatAssignment(tasks), maxTasks - minTasks < 2);
+            assertNotEquals(maxConnectors, 0, "Found no connectors running!");
```

Comment:
it should be `assertNotEquals(0, maxConnectors, "Found no connectors running!");`